tests/test_ir.py

```diff
@@ -4,479 +4,470 @@
 
 import numpy as np
 
-from devito import Eq, Grid, Function, TimeFunction, Dimension  # noqa
+from devito import Eq, Inc, Grid, Function, TimeFunction, Operator, Dimension  # noqa
 from devito.ir.equations import DummyEq, LoweredEq
 from devito.ir.equations.algorithms import dimension_sort
 from devito.ir.iet.nodes import Conditional, Expression
+from devito.ir.iet.utils import retrieve_iteration_tree
 from devito.ir.support.basic import IterationInstance, TimedAccess, Scope
 from devito.ir.support.space import (NullInterval, Interval, IntervalGroup,
                                      Any, Forward, Backward)
 from devito.ir.support.utils import detect_flow_directions
 from devito.symbolics import indexify
 
 
-@pytest.fixture(scope="session")
```

Comment:
all changes in this file are just an attempt at better organising tests. Eventually, I'm adding one new test (see below)